ide/refactoring.api/src/org/netbeans/modules/refactoring/spi/impl/RefactoringPanel.java

```diff
@@ -784,6 +815,29 @@ public void run() {
                                                     root.setNodeLabel(description + getErrorDesc(0, occurrences, hiddenOccurrences, isQuery && sizeIsApproximate.get()));
                                                     if (last) {
                                                         tree.repaint();
+                                                        if (previewButton.isSelected()) {
+                                                            if (showParametersPanel) {
+                                                                splitPane.setDividerLocation(0.3);
+                                                                if (size.get() < MAX_ROWS) {
+                                                                    expandAll();
+                                                                    if (isQuery) {
+                                                                        selectNextUsage();
+                                                                    }
+                                                                } else {
+                                                                    expandButton.setSelected(false);
+                                                                }
+                                                            } else {
+                                                                splitPane.setDividerLocation(0.3);
```

Comment:
those two `splitPane.setDividerLocation(0.3);` are probably not needed since it was already set. This could get annoying that the divider moves every time a button on the toolbar is hit.